nisystemlink/clients/dataframe/models/_api_info.py

```diff
@@ -0,0 +1,19 @@
+from nisystemlink.clients.core._uplink._json_model import JsonModel
+
+
+class Operation(JsonModel):
```

Comment:
Do we need to add these classes to [\_\_init\_\_.py](https://github.com/ni/nisystemlink-clients-python/blob/0b2bf8e8df6290cdba62a2538ebd0259718edd55/nisystemlink/clients/dataframe/__init__.py)?

Reply:
Currently, they'll be available under the `models` namespace, e.g:

`from nisystemlink.clients.dataframe.models import ApiInfo`

Do we want to make them available from the `dataframe` namespace?

Reply:
Oh, I missed the `from ._api_info import *` in the model's \_\_init\_\_.py. Yeah, that should be fine.